perun-wui-core/src/main/java/cz/metacentrum/perun/wui/model/ColumnProvider.java

```diff
@@ -139,7 +139,7 @@ public void update(int i, T t, C c) {
 	 * @param fieldUpdater updater called when user clicks on cell
 	 * @return PerunColumn displaying string value
 	 */
-	public static <T extends JavaScriptObject> PerunColumn<T, String> createColumn(PerunColumnType type, final GetValue<T, String> getter, final FieldUpdater<T, String> fieldUpdater) {
+	public static <T extends JavaScriptObject, V> PerunColumn<T, String> createColumn(PerunColumnType type, final GetValue<T, String> getter, final FieldUpdater<T, String> fieldUpdater) {
```

Comment:
Why is there `V` in type definition, if it's not used in a method ? We have specific String type used in this method as cell value type. There is another method below with <T, C> types, where String is replaced with C for generic usage.

Reply:
I have forgot to remove it.